### ISSUE TITLE
No need to wait after Wire.requestFrom().

### ISSUE THREAD

**Issue by Koepel** [NikolaiRadke/NOKO#37]
In some files, there is a `while(Wire.available()` after the Wire.requestFrom(). All of those can be removed.
Explanation: [Common-mistakes#1](https://github.com/Koepel/How-to-use-the-Arduino-Wire-library/wiki/Common-mistakes#1)

I noticed them in the files:
- NOKO/src/V10/tools/NOKO_Diagnostics/NOKO_Diagnostics.ino
- NOKO/src/V20/tools/NOKO_Diagnostics/NOKO_Diagnostics.ino
- NOKO/src/V10/English/NOKO/Si4703.cpp
- NOKO/src/V10/Deutsch/NOKO/Si4703.cpp
- NOKO/src/V20/English/NOKO/Si4703.cpp
- NOKO/src/V20/Deutsch/NOKO/Si4703.cpp

**Comment by NikolaiRadke** [comment]
Thanks for the advice. 
I removed the obsolete code and checked the other common mistakes mentioned in your link!

**Comment by NikolaiRadke** [comment]
No, i am not only checking the common mistakes, i am reading your complete wiki. 
Very handy!
